### PR TITLE
QR Code command and imageParameter improvements 

### PR DESCRIPTION
**Pull request by perronosaurio**
#850 
image
image

### CODE REVIEW

File: src/commands/misc/qrcode/read.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const fetch = require('node-fetch')
+
+module.exports = class QRCodeRead extends Command {
+  constructor (client) {
+    super(client, {
+      name: 'read',
+      aliases: ['r'],
+      parentCommand: 'qrcode',
+      parameters: [{
+        type: 'string', full: true, missingError: 'commands:qrcode.subcommands.read.noImage'
```
Comment (metehus) [inline src/commands/misc/qrcode/read.js]: use image parameter!!!!!

File: src/commands/misc/qrcode/read.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      }]
+    })
+  }
+
+  async run ({ t, channel, author, message }, text) {
+    const body = await fetch(`http://api.qrserver.com/v1/read-qr-code/?fileurl=${message.attachments.first() ? message.attachments.first().url : text}`).then(res => res.json())
```
Comment (metehus) [inline src/commands/misc/qrcode/read.js]: image parameter!

File: src/commands/misc/qrcode/read.js (hunk 1 above)
Comment (perronosaurio) [reply src/commands/misc/qrcode/read.js]: i cant wtf

File: src/commands/misc/qrcode/read.js (hunk 2 above)
Comment (pedrofracassi) [reply src/commands/misc/qrcode/read.js]: Yes for god's sake use an image parameter

**Pull request merged**